docs/getting_started.rst

```diff
@@ -75,6 +75,21 @@ The ``fuse=False`` option is often necessary if there is no support for FUSE.
 FUSE is only necessary for loading remote http datasets which are not hosted
 on a HiGlass server.
 
+If you can't open a port on your firewall, an alternative is to set up a proxy
+that will forward the requests to HiGlass. For example, you can have `jupyter-server-proxy <https://github.com/jupyterhub/jupyter-server-proxy>`_
+forward all requests based at ``http://REMOTE_IP/proxy/HG_PORT/`` to a local HiGlass
+instance listening at ``HG_PORT``. ``HG_PORT`` can be specified by the
+``server_port`` option or chosen randomly by HiGlass. You can then tell HiGlass
+to use the proxy:
+
+.. code-block:: python
+
+    higlass.display(
+        ...,
+        fuse=False,
+        proxy_base=f"http://{REMOTE_IP}/proxy/{{port}}" # {port} will be replaced by the local HiGlass port
```

Comment:
Is it on purpose that `port` is in double brackets and `REMOTE_IP` is in single brackets?

Reply:
Oh, I guess you have `REMOTE_IP` defined at the top level and want to replace it in this string and then `{{port}}` should turn into `{port}` so that it is interpolated within `higlass.display`?

Reply:
Exactly. Maybe I should insert the `REMOTE_IP` variable another way to make this clearer?

Reply:
yeah, might be clearer to just do something like:

```
proxy_base="http://my.remote.ip/proxy/{port}" # {port} will be replaced by the local HiGlass port
```

Reply:
That would eliminate the two-level f-string and leave just `{port}` being replaced which is described in the comment

Reply:
Done

Reply:
Thank you kindly!